### ISSUE TITLE
a11y issues with VoiceOver and Chrome/Safari

### ISSUE THREAD

**Issue by anezthes** [vaadin/vaadin-select#221]
Suffers from a few a11y issues:

* VoiceOver will not pronounce the label on Chrome and Safari.

* VoiceOver will not pronounce the placeholder on Chrome and Safari.

* VoiceOver will pronounce `Toggle, group` on Chrome and Safari. 'Toggle' is the toggle button's `aria-label`. Unsure where 'group' originates from.

The culprits are:

* The `input` element has `aria-hidden="true"` and `tabindex="-1"` meaning it will never be focused and the label and placeholder will never be pronounced.

* The web component itself and `vaadin-select-text-field` has `tabindex="0"`. If these are to serve as our focus elements they need to manage the label and placeholder as well so it's pronounced by ATs.

---

Additional issues:

* Allow setting `aria-label` when there's no label. Related: https://github.com/vaadin/vaadin-text-field/issues/454.

* Issues related to vaadin-list-box and vaadin-item (proposed fix is here: https://github.com/vaadin/vaadin-list-box/issues/50).

**Comment by web-padawan** [comment]
Note: `aria-hidden` was added at #18
